ui/job-view/pushes/Push.jsx

```diff
@@ -305,14 +304,32 @@ class Push extends React.Component {
 
     try {
       const decisionTaskId = await getGeckoDecisionTaskId(push.id, repoName);
-      const jobList = await RunnableJobModel.getList(repoName, {
-        decision_task_id: decisionTaskId,
-      });
+      const url = `https://queue.taskcluster.net/v1/task/${decisionTaskId}/runs/0/artifacts/public/runnable-jobs.json`;
       const { id } = push;
-
-      jobList.forEach(job => {
-        job.push_id = id;
-        job.id = escapeId(job.push_id + job.ref_data_name);
+      const headers = new Headers();
+      headers.append('x-taskcluster-skip-cache', 'true');
+
+      let rawJobs = await fetch(url, headers).then(response => response.text());
```

Comment:
Let's use `.json()` instead of `.text()` to save needing to `JSON.parse()` manually :-)